cinema/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    duration = models.IntegerField()
+
+    class Meta:
+        verbose_name_plural = "movies"
+
+    def __str__(self):
+        return f"Movies: {self.title}"
```

Comment:
The `__str__` method should return a more concise representation. Consider changing the return statement to `return self.title` to avoid redundancy, as the class name is already known when printing the object.